kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -297,7 +300,31 @@ showRule configNode = do
                 . Graph.inn'
                 . Graph.context graph
                 $ node'
+            let mrule = getRewriteRuleFromLabel
+                            . Graph.inn'
+                            . Graph.context graph
+                            $ node'
+            case mrule of
+                (Just rule) -> do
+                    let mid = getRuleIndex
+                                . Attribute.identifier
+                                . Rule.attributes
+                                . Rule.getRewriteRule
+                                $ rule
+                    putStrLn' $ maybe
+                        "Error: identifier attribute wasn't initialized."
+                        (axiomOrClaim (length axioms))
+                        mid
+                Nothing ->
+                    putStrLn' "No rule was applied."
         else putStrLn' "Invalid node!"
+  where
+    axiomOrClaim :: Int -> Int -> String
+    axiomOrClaim len iden =
+        if iden < len
```

Comment:
We prefer guards to if/then/else whenever possible:
```
  | iden < len = "Rule is axiom " <> show iden
  | otherwise = "Rule is claim " <> show (iden - len)
```